packages/griffecli/src/griffecli/__init__.py

```diff
@@ -0,0 +1,27 @@
+# This top-level module imports all public names from the CLI package,
+# and exposes them as public objects.
+
+"""Griffe CLI package.
+
+The CLI (Command Line Interface) for the griffe library.
+This package provides command-line tools for interacting with griffe.
+
+## CLI entrypoints
+
+- [`griffecli.main`][]: Run the main program.
+- [`griffecli.check`][]: Check for API breaking changes in two versions of the same package.
+- [`griffecli.dump`][]: Load packages data and dump it as JSON.
+- [`griffecli.get_parser`][]: Get the argument parser for the CLI.
```

Comment:
I see a mix of cross-ref, some using `griffe.*`, some using `griffecli.*` or `griffelib.*`, we should make them consistent.